src/index.js

```diff
@@ -5,16 +5,28 @@ const Groq = require("groq-sdk");
 const bot = new Telegraf(process.env.BOT_TOKEN);
 const groq = new Groq();
 
-bot.start((ctx) => ctx.reply("Welcome to your AI Assistant!"));
+bot.start((ctx) => ctx.reply("Witaj, hehe, jestem Seba, jak mogę Ci pomóc?"));
 
 // Handle messages and respond using Groq SDK
 bot.on("text", async (ctx) => {
   const userMessage = ctx.message.text;
 
   try {
-    // Use Groq's chat completion to generate a response
+    // Use Groq's chat completion to generate a response with a custom prompt
     const chatCompletion = await groq.chat.completions.create({
```

Comment:
**praise**: The system prompt is well-crafted, clearly defining the assistant's personality and style. This will help ensure consistent and engaging interactions.